.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,10 @@
-!!!!! STOP AND READ !!!!!
-
-If the dropdown above says "base fork: lord/master", you are submitting your change to ALL USERS OF SLATE, not just your company. This is probably not what you want. Click "base fork" to change it to the right place.
-
-If you're actually trying to submit a change to upstream Slate, please submit to our dev branch, PRs sent to the master branch are generally rejected.
+<!--
+(Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)
```

Comment:
Click the link above? where is this link?

Reply:
do flytbase have contribution guidelines already?